src/app/template-list/template-list.component.html

```diff
@@ -3,7 +3,7 @@
   <!-- you mean here candice :troll: -->
   <div class="cursor-point card-container" *ngFor="let tmpl of tmplList" (click)="chooseTmpl(tmpl)">
     <div class="card">
-      <div class="icon" (click)="removeTemplate()">
+      <div class="icon" (click)="removeTemplate(); $event.stopPropagation()">
```

Comment:
need to pass in tmpl argument to removeTemplate(), just like chooseTempl(tmpl) above